### DISCUSSION TITLE
Idea looking for coder: a plugin for the script to check for sales

### DISCUSSION THREAD

**Discussion by nappyapp** [UltimaHoarder/UltimaScraper#1278] [Ideas]
I had an idea for a tool and figured I'd put it out there and see if anyone wants to try it.

A plugin for the script, like the renamer etc, that checks for expired models who currently have a sale. Maybe even display how many new posts since expiration.

Would be useful in determining which are cost effective to resubscribe.

**Comment by Nick05491** [comment]
+1 for this.  Nowadays OF does send notifications when expired follows are offering a promotion.  Would be great if it could also check user Lists for any promotions.  I keep lists of accounts I'd like to rip, and occasionally check them for Free promotions.  Bonus if the functionality could filter based on "% off" or "promo price" to weed-out normally Free accounts and promos with only a reduced price (not free).
